Deploy-CAM.ps1

```diff
@@ -2812,6 +2909,56 @@ else {
         }
     } while ( -not $domainAdminCredential )
 
+    $radiusConfig = @{
+        enableRadiusMfa = $enableRadiusMfa
+        radiusServerHost = $radiusServerHost
+        radiusServerPort = $radiusServerPort 
+        radiusSharedSecret = $radiusSharedSecret
+    }
+    if ( ($enableRadiusMfa -eq $null) -or $enableRadiusMfa ) {
+        if ( $enableRadiusMfa -eq $null -and (-not $ignorePrompts) ) {
+            $enableRadiusMfa = (Read-Host "Do you want to enable Multi-Factor Authentication using your Radius Server? (yes/no)") -like "*y*"
+        } elseif ( $enableRadiusMfa -eq $null -and $ignorePrompts ) {
+            $enableRadiusMfa = $false
+        }
+
+        if ($enableRadiusMfa) {
+            if (-not $radiusConfig.radiusServerHost ) {
+                $radiusConfig.radiusServerHost = Read-Host "Please enter your Radius Server's Hostname or IP"
+            }
+
+            do {
+                if (-not $radiusConfig.radiusServerPort ) {
+                    try {
+                        $radiusConfig.radiusServerPort = [int](Read-Host  "Please enter your Radius Server's Listening port")
+                    } catch {
+                        $radiusConfig.radiusServerPort = $null
+                        Write-Host "Selected port is not an Integer"
+                    }
+                    if ( ($radiusConfig.radiusServerPort -le 0) -or ($radiusConfig.radiusServerPort -gt 65535) ) {
+                        Write-Host "Selected port is invalid. Should be between 1 and 65535."
+                        $radiusConfig.radiusServerPort = $null
+                    } else {
+                        $radiusConfig.radiusServerPort = $radiusConfig.radiusServerPort
+                    }
+                }
+            } while (-not $radiusConfig.radiusServerPort )
+
+            if (-not $radiusConfig.radiusSharedSecret ) {
+                $radiusConfig.radiusSharedSecret = Read-Host -AsSecureString "Please enter your Radius Server's Shared Secret"
+            }
+        }
+        $radiusConfig.enableRadiusMfa = $enableRadiusMfa
+    } 
+    
+    if ( -not $enableRadiusMfa) {
+        # Placeholder value for the radius secret and port is required in order to create KeyVault entry
+        $radiusConfig.radiusSharedSecret = ConvertTo-SecureString "radiusSecret" -AsPlainText -Force
+        $radiusConfig.radiusServerPort = ConvertTo-SecureString 0 -AsPlainText -Force
+        
+        $radiusConfig.radiusServerHost = ConvertTo-SecureString "radiusServer" -AsPlainText -Force
+    }
```

Comment:
Should we be explicitly setting $radiusConfig.enableRadiusMfa to 'false' here in this block?

Reply:
Oh I see we're doing that now :)